Front/src/components/layouts/Header.tsx

```diff
@@ -103,58 +124,58 @@ function Header() {
                                 </NavLink>
                             </li>
-                            <li>
+                            <li onClick={() => setShowResponsiveNav(false)}>
                                 <NavLink to={'contact'} className='relative after:transition-all after:absolute after:left-[50%] after:translate-x-[-50%] after:bottom-[-3px] after:w-0 after:h-0 after:bg-gray-900 hover:after:w-[100%] hover:after:h-[1.5px]'>
                                     Contact
                                 </NavLink>
                             </li>
-                            <li>
+                            <li onClick={() => setShowResponsiveNav(false)}>
                                 <NavLink to={"about"} className='relative after:transition-all after:absolute after:left-[50%] after:translate-x-[-50%] after:bottom-[-3px] after:w-0 after:h-0 after:bg-gray-900 hover:after:w-[100%] hover:after:h-[1.5px]'>
                                     About
                                 </NavLink>
                             </li>
-                            <li>
+                            <li onClick={() => setShowResponsiveNav(false)}>
                                 <NavLink to={"signup"} className='relative after:transition-all after:absolute after:left-[50%] after:translate-x-[-50%] after:bottom-[-3px] after:w-0 after:h-0 after:bg-gray-900 hover:after:w-[100%] hover:after:h-[1.5px]'>
                                     Sign Up
                                 </NavLink>
                             </li>
-                            <li>
-                                <NavLink to={"wishlist"} className='relative after:transition-all after:absolute after:left-[50%] after:translate-x-[-50%] after:bottom-[-3px] after:w-0 after:h-0 after:bg-gray-900 hover:after:w-[100%] hover:after:h-[1.5px]'>
-                                    Wishlist
-                                </NavLink>
-                            </li>
-                            <li>
-                                <NavLink to={"cart"} className='relative after:transition-all after:absolute after:left-[50%] after:translate-x-[-50%] after:bottom-[-3px] after:w-0 after:h-0 after:bg-gray-900 hover:after:w-[100%] hover:after:h-[1.5px]'>
-                                    Cart
-                                </NavLink>
-                            </li>
                         </ul>
                     </nav>
+
+                    {/* search box */}
+                    <div className="flex items-center justify-center w-full">
+                        <div className="w-full">
+                            <div className="flex w-full">
+                                <div className="flex items-center justify-center rounded-tl-lg rounded-bl-lg border-r border-gray-200 bg-white p-8">
+                                    <svg viewBox="0 0 20 20" aria-hidden="true" className="pointer-events-none absolute w-5 fill-gray-500 transition">
```

Comment:
you could make this SVG separate and import it  for more clean code

Reply:
okay i will do it , thanks for explain